api/http/routes/sms.route.go

```
@@ -20,7 +21,9 @@ func SmsHandler(c *gin.Context) {
 	if err := c.BindJSON(&reqBody); err != nil {
 		c.String(http.StatusBadRequest, err.Error())
 	}
+	ctx := context.WithoutCancel(c.Request.Context())
 	go core.NewCore().SmsUsecase.Send(
```

golangci-lint (2.12.2)

[error] Error return value of `.Send` is not checked

(errcheck)